desktop/Makefile.am

```diff
@@ -25,23 +25,109 @@
+hicolor64x64dir = $(HICOLORDIR)/64x64/apps
+hicolor128x128dir = $(HICOLORDIR)/128x128/apps
+hicolor256x256dir = $(HICOLORDIR)/256x256/apps
+hicolor512x512dir = $(HICOLORDIR)/512x512/apps
+hicolorscalabledir = $(HICOLORDIR)/scalable/apps
+
+hicolor16x16_DATA = $(ICON_SRC)/16x16/org.fontforge.FontForge.png
+hicolor22x22_DATA = $(ICON_SRC)/22x22/org.fontforge.FontForge.png
+hicolor24x24_DATA = $(ICON_SRC)/24x24/org.fontforge.FontForge.png
+hicolor32x32_DATA = $(ICON_SRC)/32x32/org.fontforge.FontForge.png
+hicolor48x48_DATA = $(ICON_SRC)/48x48/org.fontforge.FontForge.png
+hicolor64x64_DATA = $(ICON_SRC)/64x64/org.fontforge.FontForge.png
+hicolor128x128_DATA = $(ICON_SRC)/128x128/org.fontforge.FontForge.png
+hicolor256x256_DATA = $(ICON_SRC)/256x256/org.fontforge.FontForge.png
+hicolor512x512_DATA = $(ICON_SRC)/512x512/org.fontforge.FontForge.png
+hicolorscalable_DATA = $(ICON_SRC)/scalable/org.fontforge.FontForge.svg
+
+deskpixsdir = $(PACKAGEROOTDIR)/pixmaps
+deskpixs_DATA = $(ICON_SRC)/pixmaps/org.fontforge.FontForge.xpm	\
+		$(ICON_SRC)/32x32/org.fontforge.FontForge.png
+
+EXTRA_DIST = \
+	2012/src/icon-16x16-apps-fontforge.svg		\
+	2012/src/icon-24x24-apps-fontforge.svg		\
+	2012/src/icon-32x32-apps-fontforge.svg		\
+	2012/16x16/org.fontforge.FontForge.png		\
+	2012/22x22/org.fontforge.FontForge.png		\
+	2012/24x24/org.fontforge.FontForge.png		\
+	2012/32x32/org.fontforge.FontForge.png		\
+	2012/48x48/org.fontforge.FontForge.png		\
+	2012/64x64/org.fontforge.FontForge.png		\
+	2012/128x128/org.fontforge.FontForge.png	\
+	2012/256x256/org.fontforge.FontForge.png	\
+	2012/512x512/org.fontforge.FontForge.png	\
+	2012/pixmaps/org.fontforge.FontForge.xpm	\
+	2012/scalable/org.fontforge.FontForge.svg	\
+	tango/src/icon-16x16-apps-fontforge.svg		\
+	tango/src/icon-24x24-apps-fontforge.svg		\
+	tango/src/icon-32x32-apps-fontforge.svg		\
+	tango/16x16/org.fontforge.FontForge.png		\
+	tango/22x22/org.fontforge.FontForge.png		\
+	tango/24x24/org.fontforge.FontForge.png		\
+	tango/32x32/org.fontforge.FontForge.png		\
+	tango/48x48/org.fontforge.FontForge.png		\
+	tango/64x64/org.fontforge.FontForge.png		\
+	tango/128x128/org.fontforge.FontForge.png	\
+	tango/256x256/org.fontforge.FontForge.png	\
+	tango/512x512/org.fontforge.FontForge.png	\
+	tango/pixmaps/org.fontforge.FontForge.xpm	\
+	tango/scalable/org.fontforge.FontForge.svg	\
+	icons-older/ffanvil32.gif			\
+	icons-older/FFbutton-32.png			\
+	icons-older/FFslug-32.png			\
+	icons-older/fontforge.png			\
+	icons-older/fontforge.xpm			\
+	icons-older/FFbutton-128.png			\
+	icons-older/FFslug-128.png			\
+	icons-older/FFslug-48.png			\
+	icons-older/fontforge.svg			\
+	org.fontforge.FontForge.appdata.xml		\
+	org.fontforge.FontForge.desktop			\
+	fontforge.xml					\
+	org.fontforge.FontForge.metainfo.xml
+
+install-data-hook: touch-icon-dir
+uninstall-hook: touch-icon-dir
+
+touch-icon-dir:
+	if test -z "$(DESTDIR)"; then $(TOUCH) $(DESTDIR)$(HICOLORDIR) || : ; fi
```

Comment:
Why is this needed?

Reply:
found that tidbit of info while looking at all sorts of desktop installs:
https://github.com/GNOME/gimp/blob/master/desktop/Makefile.am
It was ADDDESKTOP earlier. This seemed a bit more descriptive.
take it out?

Reply:
Yeah let's keep it simple and not do this